Cargo.toml

```diff
@@ -35,6 +35,8 @@ clap = "4.5.21"
 ctrlc = { version = "3.4.5", features = ["termination"] }
 # bootstrap_srv for signature verification.
 ed25519-dalek = "2.1.1"
+# automock traits for tests
+mockall = "0.13"
```

Comment:
move down to dev dependencies?

Reply:
all the mockall feature gating https://github.com/holochain/kitsune2/pull/63/commits/d123d56c507e42f05e2088fa03d4da9c6b452e86